docs/design/vocabulary.md

```diff
@@ -39,25 +42,52 @@ asset Truck extends Vehicle {
 locale: en
 namespace: org.acme
 declarations:
-  - Color: A color
-  - Vehicle: A road vehicle
-    properties:
-      - vin: Vehicle Identification Number
-      - model: Model of the vehicle
-  - Truck: A vehicle capable of carrying cargo
-    properties:
-      - weight: The weight of the truck in KG
+  - Color:
```

Comment:
This vocabulary yam file is incorrect. Please refer to the vocabulary file here.
https://github.com/accordproject/concerto/pull/664/files#diff-ebc9ab70877a4dfb90adb68e9c70c0532b0c79567a6b96ff4761a2df1071f6f3